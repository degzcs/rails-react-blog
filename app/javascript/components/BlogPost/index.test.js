import React from "react";
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router'

import BlogPost from "./index.jsx";

test('BlogPost show data', () => {
  const post = {
		title: 'test',
		description: 'test',
    slug: 'test',
    image: 'http://image.jpg'
	}

  const component = renderer.create(
    <StaticRouter location="someLocation">
      <BlogPost post={post}/>
    </StaticRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
