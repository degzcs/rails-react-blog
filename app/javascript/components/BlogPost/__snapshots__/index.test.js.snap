// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost show data 1`] = `
<a
  href="/blog/test"
  onClick={[Function]}
>
  <div
    className="col-12 mt-3"
  >
    <div
      className="bcard"
    >
      <div
        className="row"
      >
        <div
          className="col-lg-5  col-sm-12 card-body"
        >
          <h3
            className="card-title"
          >
            test
          </h3>
          <p
            className=""
          >
            test
          </p>
        </div>
        <div
          className="col-lg-7 col-sm-12 image-bk"
          style={
            Object {
              "backgroundImage": "url(http://image.jpg)",
              "backgroundPosition": "50% 50%",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "100%",
              "paddingBottom": "25%",
            }
          }
        />
      </div>
    </div>
  </div>
</a>
`;
